website/docs/tech-explanation/ckb-address.mdx

```diff
@@ -0,0 +1,65 @@
+---
+id: ckb-address
+title: CKB Address
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+CKB address packages the lock script into a single line of verifiable and human-readable format, adhering to Bitcoin's [Bech32m address format (BIP-350)](https://github.com/sipa/bips/blob/bip-bech32m/bip-0350.mediawiki). In this format, the lock script consists of three key parameters: `code_hash`, `hash_type`, and `args`.
+
```

Comment:
Use "Lock Script" instead of "lock script" as it's a term in CKB?

Reply:
updated